pyproject.toml

```diff
@@ -54,4 +54,3 @@ build-backend = "poetry.core.masonry.api"
 
 [tool.pytest.ini_options]
 pythonpath = "gpbp"
-addopts = "--cov=gpbp"
```

Comment:
I also had to remove this line in order to run the debugger, otherwise it wouldn't work. But I decided not to commit this change because I wasn't sure how this affects the code coverage reports.

Reply:
If we remove this line, then the coverage report doesn’t get automatically generated when you run tests. 

Since it’s hindering development, let’s just remove it. We can run the report in other ways and/or add this line in the future.

Reply:
@pkollenda: FYI